src/editor/AdvancedSidebar.js

```diff
@@ -17,18 +17,27 @@ import { CategoryAutocomplete } from 'newspack-components';
 const AdvancedSidebar = ( { onMetaFieldChange, excluded_categories = [], excluded_tags = [] } ) => {
 	return (
 		<Fragment>
-			<BaseControl className="newspack-popups__segmentation-sidebar">
+			<BaseControl className="newspack-popups__advanced-sidebar">
```

Comment:
Where is this class styled? I can't see anything when searching for `__advanced`.

Reply:
Oh, I originally added some CSS to create some space in between the two fields, but decided that the `<hr />` looked better. Regardless, the class name `__segmentation-sidebar` is inaccurate and potentially confusing.

Reply:
But the `.newspack-popups__advanced-sidebar` is not styled, so there's no reason for any `className` prop here.

Reply:
Removed in 8bd6b66